node/lib/DeviceAccessor.ts

```diff
@@ -1,18 +1,107 @@
 
 import { DeviceInterface } from "./DeviceInterface";
-import { IDevice } from "./ITranslator";
+import { ITranslator } from "./ITranslator";
 
 import { EventEmitter } from "events";
 
 /**
  * Provides reflection-style access to device properties and methods via device interfaces.
  */
 export class DeviceAccessor {
```

Comment:
> DeviceAccessor { [](start = 12, length = 17)

So, will we will be adding projections around DeviceAccessor in opent2t\client repo, for apps to consume ?